apps/mobile/convex/confect/http-api.ts

```
@@ -0,0 +1,150 @@
+        // TODO: Fix HTTP API integration with Confect functions
+        return [];
+      })
+    )
+);
+
+const SessionsApiGroupLive = HttpApiBuilder.group(OpenAgentsApi, "sessions", (handlers) =>
+  handlers.handle("getPendingSessions", () =>
+    Effect.gen(function* () {
+      // TODO: Fix HTTP API integration with Confect functions
+      return [];
+    })
```

Biome (2.1.2)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)